### PR TITLE
[9.0] Port partner_contact_nationality.

### PR DESCRIPTION
**Pull request by richard-willdooit**

### CODE REVIEW

File: partner_contact_nationality/README.rst [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
-Module name
-===========
+===================
+Partner Nationality
+===================
 
-This module was written to extend the functionality of Odoo to support setting
-a nationality for your contacts.
+This module was written to extend the contact management functionality. It
```
Comment (elicoidal) [inline partner_contact_nationality/README.rst]: Hi @richard-willowit thanks for the contribution.
This module was written to extend => This module extends

**Comment by elicoidal** [comment]
LGTM

**Comment by moylop260** [comment]
:+1:

**Comment by Cedric-Pigeon** [comment]
:+1:

**Pull request merged**